sources/packages/backend/apps/api/src/route-controllers/overaward/models/overaward.dto.ts

```diff
@@ -30,7 +30,7 @@ export class StudentsOverawardAPIOutDTO {
   assessmentTriggerType?: AssessmentTriggerType;
 }
 
-export class AESTOverawardAPIOutDTO extends StudentsOverawardAPIOutDTO {
+export class OverawardDetailsAPIOutDTO extends StudentsOverawardAPIOutDTO {
```

Comment:
Can you please explain why this was renamed?

Reply:
Initially, while creating the `overaward.institutions.controller.ts` I did not realize that `addedByUser` does not need to be returned for the institution. Therefore, I changed it to a common dto. But now, I understand that `addedByUser` is not supposed to be returned for `institutions` and is only specific to `AEST`, therefore reverting it back.